### ISSUE TITLE
Enable ASLR when building dlls

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3609]
**Reported by tspivey on 2013-10-28 02:59**
Since NVDA injects code into other processes, I think ASLR should be enabled for its dlls. Linking with /dynamicbase should do it. Pre-vista systems should ignore this flag; I haven't yet tried this to see what side effects this might have.

**Comment by leonardder** [comment]
CC @tspivey @JCSTeh

**Comment by michaelDCurran** [comment]
@tspivey for completeness, could you please outline why this is necessary?

**Comment by Adriani90** [comment]
@tspivey could you please reply to the comment above? Thanks.

**Comment by leonardder** [comment]
As per above, I"m closing this as works for me. If this issue requires a revisit, let us now!
